lib/kubeclient/version.rb

```diff
@@ -1,4 +1,4 @@
 # Kubernetes REST-API Client
 module Kubeclient
-  VERSION = '2.3.0'.freeze
+  VERSION = '2.3.1'.freeze
```

Comment:
This PR adds new functionality, and as such, per semantic versioning, the minor version of the gem should be bumped, not the patch. (in other words, the middle digit, not the last one)
Also, not after every PR a gem version is bumped, so in most of the cases, the version is not bumped as part of the PR.